app/assets/javascripts/backbone/views/timeline_table_view.coffee

```diff
@@ -22,7 +27,37 @@ class MS.Views.TimelineTableView extends Backbone.View
     e.preventDefault()
     @modal.show()
 
+  newChildGroup: (e) ->
+    e.preventDefault()
+
+    $project_group = $(e.target).closest(".project-group")
+    project_group_id = $project_group.data("action-key")
```

Comment:
I saw that being used in `batch_actions` and thought I would copy the same name. I do like `id` better, though.